lib/bokken_web/controllers/lecture_controller.ex

```diff
@@ -11,7 +12,8 @@ defmodule BokkenWeb.LectureController do
     render(conn, "index.json", lectures: lectures)
   end
 
-  def create(conn, %{"lecture" => params}) when is_map_key(params, "assistant_mentors") do
+  def create(conn, %{"lecture" => params})
+      when is_map_key(params, "assistant_mentors") and is_organizer(conn) do
```

Comment:
I'm not sure about this. @ruioliveira02 Can I hear your opinion? 🙏🏼
Same for the other endpoints of this controller.

Reply:
With our current workflow, only organizers should be able to create lectures.